### PR TITLE
release for v0.1.0-rc2

### PR DESCRIPTION
**Pull request by ibrokethecloud**
PR contains release for seeder chart v0.1.0-rc2

since seeder needs `harvester-system` namespace to exist the CI has an additional step to create `harvester-system` namespace.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -56,6 +56,10 @@ jobs:
         uses: helm/kind-action@v1.2.0
         if: steps.list-changed.outputs.changed == 'true'
 
+      - name: Create harvester-system namespace
+        run: "kubectl create namespace harvester-system"
+        if: steps.list-changed.outputs.changed == 'true'
```
Comment (guangbochen) [inline .github/workflows/ci.yaml:61]: we probably don't need this since the helm test will create the deploy ns automatically.

Comment (ibrokethecloud) [reply .github/workflows/ci.yaml:61]: it doesn't seem to. By default it deploys charts to a randomly created namespace.

https://github.com/harvester/charts/actions/runs/5459186904/jobs/9934917079#step:10:2107

seeder chart on initialization is looking to define a `local` cluster object in the harvester-system namespace which is why it fails.

**Review by guangbochen — APPROVED** [review]
LGTM thanks, and do we need to backport https://github.com/harvester/charts/pull/188/commits/a98e7939a694e3fb60441626e87e36ae14acc00a to the master?

**Comment by ibrokethecloud** [comment]
I will create another PR to backport this.

**Review by bk201 — APPROVED** [review]
LGTM!

**Pull request merged**